docs/guides/quick-start/miranum-ide/create-form.md

```diff
@@ -0,0 +1,87 @@
+---
+id: miranum-ide-create-form-quickstart
+title: Create your Form
+sidebar_label: Create your Form
+description: "This section introduces Miranum Form Builder as the tool to create and edit forms for Miranum Platform."
+---
+
+import ElementPalette from "../static/img/miranumIDE/miranumIDE_form-element-palette.png"
+import General from "../static/img/miranumIDE/miranumIDE_form-general.png"
+import Options from "../static/img/miranumIDE/miranumIDE_form-options.png"
+import Validation from "../static/img/miranumIDE/miranumIDE_form-validation.png"
+import KeyField from "../static/img/miranumIDE/miranumIDE_form-key-field.png"
+import LinkToBpmn from "../static/img/miranumIDE/miranumIDE_form-link-to-bpmn.png"
+
+# Miranum Form Builder Quickstart Guide
+
+**Miranum Form Builder** is one component of the Miranum IDE.  
```

Comment:
⛏️ - We should clearly state that the Miranum Form Builder can be only used in case one also uses Miranum Platform.

Reply:
It might also be worth mentioning the differences between the Form Builder and the JSON Forms Plugin

Reply:
I added some info sections for that.